src/go/cmd/content.go

```diff
@@ -57,14 +57,14 @@ func (c content) isEmpty() bool {
 }
 
 // adds const and var declaration to the exports list
-func (c *content) addGenDecl(pkg Pkg, tok token.Token, vs *ast.ValueSpec) Declaration {
+func (c *content) addGenDecl(pkg Pkg, tok token.Token, vs *ast.ValueSpec, imports map[string]string) Declaration {
```

Comment:
The `imports` param appears to be optional.  If this is the case, it would be good to add a comment explaining when it's needed (applies to all affected `content` methods).